etcdctl/command/handle.go

```diff
@@ -130,15 +130,27 @@ func printAll(resp *etcd.Response, format string) {
 	if resp.Node.Dir == true {
 		return
 	}
-	printKeyOnly(resp, format)
+	printKeyOnly(resp, format, false)
+}
+
+// printAllPrependKey is just like printAll, but prepends the key name to the new value
+func printAllPrependKey(resp *etcd.Response, format string) {
+	if resp.Node.Dir == true {
+		return
+	}
+	printKeyOnly(resp, format, true)
 }
 
 // printKeyOnly only supports to print key correctly.
-func printKeyOnly(resp *etcd.Response, format string) {
+func printKeyOnly(resp *etcd.Response, format string, prependKey bool) {
 	// Format the result.
 	switch format {
 	case "simple":
-		fmt.Println(resp.Node.Value)
+		if prependKey {
+			fmt.Println(fmt.Sprintf("%s: %s", resp.Node.Key, resp.Node.Value))
```

Comment:
I think we can drop the extra call to `fmt.Sprintf` and just use `fmt.Printf` instead of `fmt.Println`.